express-graphql/express-graphql.d.ts

```diff
@@ -8,45 +8,48 @@
 declare module "express-graphql" {
     import { Request, Response } from "express";
 
-    /**
-     * Used to configure the graphQLHTTP middleware by providing a schema
-     * and other configuration options.
-     */
-    export type Options = ((req: Request) => OptionsObj) | OptionsObj
-    export type OptionsObj = {
-        /**
-         * A GraphQL schema from graphql-js.
-         */
-        schema: Object,
-
-        /**
-         * An object to pass as the rootValue to the graphql() function.
-         */
-        rootValue?: Object,
-
+    namespace graphqlHTTP {
         /**
-         * A boolean to configure whether the output should be pretty-printed.
+         * Used to configure the graphQLHTTP middleware by providing a schema
+         * and other configuration options.
          */
-        pretty?: boolean,
-
-        /**
-         * An optional function which will be used to format any errors produced by
-         * fulfilling a GraphQL operation. If no function is provided, GraphQL's
-         * default spec-compliant `formatError` function will be used.
-         */
-        formatError?: Function,
-
-        /**
-         * A boolean to optionally enable GraphiQL mode.
-         */
-        graphiql?: boolean,
-    };
-
-    type Middleware = (request: Request, response: Response) => void;
+        export type Options = ((req:Request) => OptionsObj) | OptionsObj
+        export type OptionsObj = {
+            /**
+             * A GraphQL schema from graphql-js.
+             */
+            schema:Object,
+
+            /**
+             * An object to pass as the rootValue to the graphql() function.
+             */
+            rootValue?:Object,
+
+            /**
+             * A boolean to configure whether the output should be pretty-printed.
+             */
+            pretty?:boolean,
+
+            /**
+             * An optional function which will be used to format any errors produced by
+             * fulfilling a GraphQL operation. If no function is provided, GraphQL's
+             * default spec-compliant `formatError` function will be used.
+             */
+            formatError?:Function,
+
+            /**
+             * A boolean to optionally enable GraphiQL mode.
+             */
+            graphiql?:boolean,
+        };
+
+        type Middleware = (request:Request, response:Response) => void;
+    }
 
     /**
      * Middleware for express; takes an options object or function as input to
      * configure behavior, and returns an express middleware.
      */
-    export default function graphqlHTTP(options: Options): Middleware;
```

Comment:
I replaced it to enable the use of ES6 syntax for imports. With the current implementation, using ES6 syntax for importing (for example `import * as graphqlHTTP from "express-graphql";`) fails with compile error `TS2349: Cannot invoke an expression whose type lacks a call signature.`. The proposed implementation will enable the use ES6 syntax to import this module in Typescript. The new implementation follows the suggestion made in [this discussion](https://github.com/Microsoft/TypeScript/issues/5565).

Suggestion:
Replace the import statement in the test file _express-graphql-tests.ts_ by
```
import * as graphqlHTTP from "express-graphql";
```